examples/utils/bundle-size-tests/package.json

```
@@ -29,6 +30,7 @@
 		"lint": "fluid-build . --task lint",
 		"lint:fix": "fluid-build . --task eslint:fix --task format",
 		"prepack": "npm run webpack",
+		"test": "mocha",
```

I don't think our CI runs this as-is. Looks like the idiomatic pattern is

```suggestion
		"test": "npm run test:mocha",
		"test:mocha": "mocha",		
```

where the first script just makes local testing more convenient (the `test:mocha` script is what's actually invoked by pr gate / ci). I'm judging this by test browser on the pipeline build + looking at other packages.

---

Good to know. I'll also need to update the fluidBuild task below, or that won't webpack before mocha.